compiler/src/typed/translprim.re

```diff
@@ -1408,23 +1420,70 @@ let transl_prim = (env, desc) => {
 
   let diable_gc = [(Location.mknoloc("disableGC"), [])];
```

Comment:
Do you mind fixing the spelling of this while you're in here?

Reply:
I thought if I did you would hit me with "this should be a separate PR" 😅 

You don't think it should be a separate one?

Reply:
Nah, because you used the value in this PR. Good thinking though 😉